src/main/java/namor/template/calculator/CalculatorFactory.java

```diff
@@ -0,0 +1,17 @@
+package namor.template.calculator;
+
+
+import namor.template.stack.ArrayStack;
+
+public class CalculatorFactory {
+
+    public Calculator getCalculator(String inputString) {
```

Comment:
words calculator in class name and in method name, i thing method name  "get" is enough